### PR TITLE
Misc. tweaks and bugfixes

### LINKED ISSUE #1527: [Bug]: error in androids mod patch
Rimworld version:
Combat Extended version: 4.6
Combat Extended source: steam

**Description**
Startup error with androids mod installed. Look like this commit https://github.com/CombatExtended-Continued/CombatExtended/commit/68dd4b523d064fa07cfcbc07895c276228cc436e change "Patches/Androids/AlienRace/AlienRace_Chj.xml" file to patch ThingDef_AlienRace "ATPP_Android2KITX" which is not part of Androids mod. It is part of "Android Tiers - TX Series" mod. (copy paste bug?)

**Expected behavior**
No error in console

**To reproduce**
Start rimworld with CE and androids mod

**Screenshots & log dumps**
Console show this error:
[Combat Extended] Patch operation Verse.PatchOperationSequence(count=1, lastFailedOperation=Verse.PatchOperationFindMod(Androids)) failed
file: D:\SteamLibrary\steamapps\workshop\content\294100\1631756268\Patches\Androids\AlienRace\AlienRace_Chj.xml

**Complete the following checklist**
I hereby verify that I have done the following:
- [x] Confirmed that my game version and load order are correct.
- [x] Confirmed that I am running the appropriate and most updated version of Combat Extended and required compatibility patches.
- [x] Confirmed I am not running any mods with known incompatibilities with Combat Extended.
- [x] Disabled Combat Extended and attempted to reproduce the behavior without success.

### LINKED ISSUE #1532: unmark grenades smeltable
## Autogenerated Issue
Name: unmark grenades smeltable
Submitted By: lperkins2
Short Description: smeltable tag should be removed or set to false for EMP grenades
Context: 
References: https://discord.com/channels/278818534069501953/324222101550661663/968734966949236776

### PR DESCRIPTION
**Pull request by ItsSamGray**
## Changes

- Fixed the VAE patch for the Prestige Siegebreaker and Trooper armors.
- Readjusted the Siegebreaker helmet variants' NV bonuses to be the same as Cataphract helmets.
- Archived the attachment box textures.
- Fixed the Megapede variants' armor value.
- Fixed the Insectoid Queen's armor value.
- Fixed the Gigalocust variants' movement speed.
- Changed the Tribal Chief apparel patch to a Conditional.
- Fixed the explosive comp for FSX and Prometheum to make them cook off again if damaged.
- Fixed #1527 
- Readjusted Gaubric from VIE-D
- Fixed #1532 
- Fixed the generic ammo for VWE-Frontier's new ammo types
- Fixed and tweaked vanilla and modded insectoids' melee attack tools
- Housekeeping for the VFE-Security mod (the flamethrower turret)
- Patched the height Defoliator building from Royalty and adjusted the Gloomlight's fillPercent

## Reasoning

- Gigalocusts are not supposed to be that fast, 4.0c/s in vanilla. 4.8 is more than enough. They have a special drop-in raid type and are hard to manage if they are too fast.
- The megapede was adjusted accordingly in my previous PR to have armor instead of incoming damage multiplier. The goal was not to nerf them as they are nerfed now.
- The Queen is very moderately armored in vanilla, it's not supposed to tank bullets, 10/22 was too high for 30%/30% .
- Each chemfuel explodes in a 1.1 tile radius with 25 of them exploding in a 5 tile radius and 75 chemfuel exploding in a 12 tile radius. Each FSX and Prometheum explode in a 0.9 and 1.9 tile radius respectively while 25 of them explode in a 5 and 7 tile radius respectively to preserve the balance. Before the previous comp was broken, the explosion radii for both FSX and Prometheum was much bigger and very absud and damaging, to the point of people abusing them for dealing with threats. These new values should be balanced enough considering how chemfuel behaves.
- The Androids patch had an extra patch with a target that was not inside the Androids mod itself which caused the patch to fail.
- Gaubric from VIE-D has been nerfed heavily in its original mod down to a normal leather, it used to be on par with devilstrand so the previous values reflected that. Now that the vanilla form of it has been nerfed, the CE patch is tweaked to account for that accordingly.
- Vanilla and modded insectoid's melee attack tools were inconsistent, fixed them with appropriate values according to the spreadsheet.
- The defoliator is as tall as autocannons and similar 2x2 building in vanilla, patched them to have the same height as them in CE as well. The mech node building (Gloomlight) is kinda the same as a standing lamp, adjusted the fillPercent to reflect that.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors

### CODE REVIEW

File: Patches/Vanilla Ideology Expanded - Dryads/ThingDefs_Items/Items_Resource_Stuff.xml [hunk 1]
```diff
@@ -27,31 +27,6 @@
...
-					</value>
-				</li>
-
-				<li Class="PatchOperationConditional">
-					<xpath>Defs/ThingDef[defName="VDE_Gaubric"]/stuffProps/categories</xpath>
-					<nomatch Class="PatchOperationAdd">
-						<xpath>Defs/ThingDef[defName="VDE_Gaubric"]/stuffProps</xpath>
-						<value>
-							<categories>
-							  <li>SoftArmor</li>
```
Comment (N7Huntsman) [inline Patches/Vanilla Ideology Expanded - Dryads/ThingDefs_Items/Items_Resource_Stuff.xml:43]: Why aren't we using it as soft armor anymore? Are we falling back to the description of it as being essentially leather?

File: Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Black_Hive.xml [hunk 2]
```diff
@@ -255,33 +255,33 @@
 							<capacities>
 								<li>Cut</li>
 							</capacities>
```
Comment (N7Huntsman) [inline Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Black_Hive.xml:257]: In the RW lore, insects were bioengineered to fight Mechanoids, thus their AP is artificially high.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
- Leave insect AP as-is so they can fight mechs.
- Clean up commented out stuff.
- Consolidate the royalty stuff into a single check.

File: Patches/Vanilla Ideology Expanded - Dryads/ThingDefs_Items/Items_Resource_Stuff.xml (hunk 1 above)
Comment (ItsSamGray) [reply Patches/Vanilla Ideology Expanded - Dryads/ThingDefs_Items/Items_Resource_Stuff.xml:43]: yea it's been nerfed down to a normal leather in the original mod

**Review by albertbokor — APPROVED** [review]
looks good, waiting for N7 to also approve

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**